docs/source/docs/index.rst

```diff
@@ -141,10 +141,17 @@ Read the research:
    ../examples/managed-jobs
    ../reference/job-queue
    ../examples/auto-failover
-   ../reference/kubernetes/index
    ../running-jobs/distributed-jobs
    ../running-jobs/many-jobs
 
+.. toctree::
+   :hidden:
+   :maxdepth: 1
+   :caption: Reservations & On-Prem
```

Comment:
To discuss: k8s page can be used for both cloud (elastic or not) & on-prem, so "on-prem" in the title doesn't fully capture it.
- Kubernetes & Reservations
- Existing Clusters & Reservations
- ...
?

Reply:
Changed to `Reservations & Kubernetes Clusters`. How does that sound?